agent/conf/agent.properties

```diff
@@ -156,3 +156,7 @@ hypervisor.type=kvm
 #router.aggregation.command.each.timeout=600
 #timeout value for aggregation commands send to virtual router
 #
+# host.overcommit.mem.mb = 0
+# allows to increase amount of ram available on host virtually to utilize Zswap, KSM features
+# and modern fast SSD/3D XPoint devices
```

Comment:
You might want to add that it adds these X amount of memory to the memory it reports to the Management Server

Reply:
@wido fixed, take a look.